common/app/views/fragments/atoms/atom.scala.html

```diff
@@ -1,11 +1,11 @@
-@(model: _root_.model.content.Atom, shouldFence: Boolean, amp: Boolean, mainMedia: Boolean)(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+@(model: _root_.model.content.Atom, shouldFence: Boolean, amp: Boolean, mainMedia: Boolean, immersiveMainMedia: Boolean)(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
 @import _root_.model.ShareLinkMeta
 @import _root_.model.content.{InteractiveAtom, MediaAtom, Quiz}
 
 @{
     model match {
         case quiz: Quiz => views.html.fragments.atoms.quiz(quiz, maybeResults = None, showResults = false, ShareLinkMeta(Nil, Nil))
-        case media: MediaAtom => views.html.fragments.atoms.media(media, amp, displayCaption = true, displayEndSlate = mainMedia, mainMedia = mainMedia)
+        case media: MediaAtom => views.html.fragments.atoms.media(media, amp, displayCaption = true, displayEndSlate = mainMedia, mainMedia = mainMedia, immersiveMainMedia = immersiveMainMedia)
```

Comment:
I think you could do `displayCaption = !immersiveMainMedia`
Then in the youtube template you don't need to do the extra check on `@if(displayCaption & !immersiveMainMedia)`